code/modules/vore/eating/living_vr.dm

```diff
@@ -25,7 +25,7 @@
 		return 1
 	M.verbs += /mob/living/proc/insidePanel
 
-	M.appearance_flags |= PIXEL_SCALE
+	//M.appearance_flags |= PIXEL_SCALE
```

Comment:
Yeah that wasn't intended to sneak into the pr at all in the first place. Only had that thing in the background for testing purposes as it kept erroring the compile.